src/actions/config.js

```diff
@@ -26,33 +25,45 @@ export function configFetchError() {
   };
 }
 
-export function asyncConfigFetch() {
+export function asyncConfigInit() {
   return dispatch => {
-    dispatch(configFetch());
+    dispatch(asyncConfigFetch('./config.js'));
+    dispatch(asyncConfigFetch('./userConfig.js'));
```

Comment:
I'm not sure if dispatch actually creates a new thread or not. Is there a possibility that `./userConfig.js` can be called before `./config.js`?